dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/DSpaceObjectUtils.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.utils;
+
+import java.sql.SQLException;
+import java.util.UUID;
+
+import org.dspace.content.DSpaceObject;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.DSpaceObjectService;
+import org.dspace.core.Context;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+
+/**
+ * Utility class providing methods to deal with generic DSpace Object of unknown type
+ *
+ * @author Mykhaylo Boychuk (4science.it)
+ */
+@Component
+public class DSpaceObjectUtils {
+
+    @Autowired
+    private ContentServiceFactory contentServiceFactory;
+
+    public DSpaceObject findDSpaceObject(Context context, UUID uuid) throws SQLException {
```

Comment:
This method should have Javadocs, please

Reply:
done, I don't know why github doesn't note it https://github.com/DSpace/DSpace/pull/2604/commits/03238bcbb5fd5303dc7b7a7a1d696096ad935391#diff-26c81a6c14a0b2489296ceec4d1dc488R33